test_data/yast/raid/raid1_encrypted_home.yaml

```diff
@@ -0,0 +1,43 @@
+<<: !include test_data/yast/raid/raid_gpt_disks_with_bios-boot.yaml
+asset_files: "/var/log/YaST2/y2log*"
```

Comment:
I don't think this is needed, that is used by other test suite linked to the scenario where you copy/paste to start this file most likely.

Reply:
Removed.